src/components/AlbumCard/Index.js

```diff
@@ -0,0 +1,35 @@
+
+import { useEffect, useState } from "react";
+import "../AlbumCard/AlbumCard.css";
+import SongList from "../SongList/Index.js";
+import { MdOutlinePlaylistPlay } from "react-icons/md";
+
+
+
+export default function AlbumCard({ album }) {
+ const [list, setList] = useState(false);
```

Comment:
for clarity, you can rename the variable to "displayList" or "showList", some programmers might assume that this state holds an array of data (because the variable sounds like it).